firefox-ios/Client/Frontend/Home/Homepage Rebuild/HomepageViewController.swift

```diff
@@ -123,21 +179,56 @@ final class HomepageViewController: UIViewController, ContentContainable, Themea
         for item: HomepageDiffableDataSource.HomeItem,
         at indexPath: IndexPath
     ) -> UICollectionViewCell {
-        // TODO: FXIOS-10163 - Dummy collection cells, to update with proper cells
-        guard let cell: UICollectionViewCell = collectionView?.dequeueReusableCell(
+        // TODO: FXIOS-10162 - Dummy collection cells, to update with proper cells
+        guard let section = HomepageSection(
+            rawValue: indexPath.section
+        ), let cell: UICollectionViewCell = collectionView?.dequeueReusableCell(
```

Comment:
making the change + using CTRL-I, it gives me this btw:
<img width="859" alt="image" src="https://github.com/user-attachments/assets/9688e51a-e039-4ea1-a482-e5e81c76a1aa">

instead of this (using CTRL-M)
<img width="816" alt="image" src="https://github.com/user-attachments/assets/4b09d591-3d49-4399-859f-79defe2c7f1a">

do you still prefer the change?

Reply:
let's go with the reident version (ctrl-I). Ctrl-M is not an indent, it's an argument break that has some weird funcitonality in certain spaces (like closures).

let's not worry about it thogh for this PR. It's a dumb thing to block it on. Merge away. :)

Reply:
sounds good! made the change anyway, its a quick one :) thanks for confirming!